### PR TITLE
chore: update channel teams action label

### PR DESCRIPTION
**Pull request by GleidsonDaniel**
## Proposed changes

Update channel teams action label

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] I have added necessary documentation (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

Jira: [TC-791]

[TC-791]: https://rocketchat.atlassian.net/browse/TC-791?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/TeamChannelsView.tsx [hunk 1]
```diff
@@ -520,10 +520,10 @@ class TeamChannelsView extends React.Component<ITeamChannelsViewProps, ITeamChan
 			return <BackgroundContainer loading />;
 		}
 		if (isSearching && !search.length) {
-			return <BackgroundContainer text={searchText ? I18n.t('No_team_channels_found') : ''} />;
```
Comment (reinaldonetof) [inline app/views/TeamChannelsView.tsx:523]: The message `No_team_channels_found` is been used just here. Could you delete it from the jsons too?

File: app/i18n/locales/en.json [hunk 2]
```diff
@@ -744,5 +744,6 @@
   "accept": "Accept",
   "Incoming_call_from": "Incoming call from",
   "Call_started": "Call started",
-  "Message_has_been_shared":"Message has been shared"
+  "Message_has_been_shared":"Message has been shared",
+  "No_channels_in_team": "No Channels on this Team"
```
Comment (reinaldonetof) [inline app/i18n/locales/en.json:748]: on this PR https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5056 was removed unnecessary uppercase, I think we should keep this behavior and change the `Team` to `team`

Comment (reinaldonetof) [reply app/i18n/locales/en.json:748]: Also, IMHO the "in" is more correct than "on" 😅

**Review by reinaldonetof — CHANGES_REQUESTED** [review]

File: app/i18n/locales/en.json (hunk 2 above)
Comment (GleidsonDaniel) [reply app/i18n/locales/en.json]: in and on have different meanings depending on the context, in this context the meaning would be more or less like this:
in: Não existem canais nesse time.
on: Esse time não tem canais.

Literally translating it sounds weird.

**Review by reinaldonetof — APPROVED** [review]

**Pull request merged**